test/src/test/java/hudson/util/ArgumentListBuilder2Test.java

```diff
@@ -66,13 +69,14 @@ public void slaveMask() throws Exception {
         args.add("java");
         args.addMasked("-version");
 
-        Slave s = j.createSlave();
-        s.toComputer().connect(false).get();
+        Slave s = j.createOnlineSlave();
+        /* TODO https://github.com/jenkinsci/jenkins-test-harness/pull/127
```

Comment:
Pending merge & release of https://github.com/jenkinsci/jenkins-test-harness/pull/127.